hack/install-go.sh

```diff
@@ -2,7 +2,8 @@
 
 destination=$1
 version=$(grep "^go " go.mod |awk '{print $2}')
-tarball=go$version.linux-amd64.tar.gz
+arch=$(uname -m | sed 's/x86_64/amd64/')
```

Comment:
lets add auto detection please ?

Reply:
Added auto detection as default mode.

Reply:
shouldn't it be ? (unless for go binary the naming are different)
`uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/'`

Reply:
Yes added for arm64.